test/tools/spec-runner/context.js

```diff
@@ -38,7 +38,8 @@ class TestRunnerContext {
       user: undefined,
       authSource: undefined,
       useSessions: true,
-      skipPrepareDatabase: false
+      skipPrepareDatabase: false,
+      requiresCSFLE: false
```

Comment:
Happy to remove this - I just made requiresCSFLE an argument that gets passed into the test runner instead of appending it later.  It was used in client_side_encryption.spec.test.ts and it looked a little cleaner.